src/test/java/uk/gov/hmcts/cmc/claimstore/events/paindinfull/PaidInFullCitizenNotificationHandlerTest.java

```diff
@@ -42,17 +45,44 @@ public void setUp() {
     }
 
     @Test
-    public void sendNotificationsSendsNotificationsToDefendant() throws NotificationClientException {
-
-        PaidInFullEvent event = new PaidInFullEvent(SampleClaim.getDefault());
+    public void sendNotificationsSendsNotificationsToDefendantEmailNotLinked() throws NotificationClientException {
+        Response fullDefenceResponse = SampleResponse.FullDefence.builder().build();
+        PaidInFullEvent event = new PaidInFullEvent(SampleClaim.getWithResponse(fullDefenceResponse));
 
         handler.notifyDefendantForPaidInFull(event);
+        verify(notificationService, once()).sendMail(
+            eq(event.getClaim().getDefendantEmail()),
+            eq(CLAIMANT_SAYS_DEFENDANT_PAID_IN_FULL_TEMPLATE),
+            anyMap(),
+            eq(referenceForDefendant(event.getClaim().getReferenceNumber()))
+        );
+    }
 
+    @Test
+    public void sendNotificationsSendsNotificationsToDefendantEmailLinked() throws NotificationClientException {
+        Response fullDefenceResponse = SampleResponse.FullDefence.builder().build();
+        PaidInFullEvent event =
+            new PaidInFullEvent(SampleClaim.getWithResponseDefendantEmailVerified(fullDefenceResponse));
+
+        handler.notifyDefendantForPaidInFull(event);
         verify(notificationService, once()).sendMail(
             eq(event.getClaim().getDefendantEmail()),
             eq(CLAIMANT_SAYS_DEFENDANT_PAID_IN_FULL_TEMPLATE),
             anyMap(),
             eq(referenceForDefendant(event.getClaim().getReferenceNumber()))
         );
     }
+
+    @Test
+    public void sendNotificationsSendsNotificationsToDefendantNoEmail() throws NotificationClientException {
```

Comment:
Please rename this test i.e `sendNotificationsDoesNotSendNotificationToDefendantWhenNoEmailAddress`